scripts/take_bias.sh

```diff
@@ -30,15 +30,15 @@ P=$1
 SAVE_FILE=${2:-"bias-%Y%m%d-%H%M%S.cr2"}
 echo "Taking bias frames on port ${P}: ${SAVE_FILE}"
 
-NUM_FRAMES=${4:-1}
-SHUTTER_INDEX=${3:-52} # 1/4000s on EOS 100D
+NUM_FRAMES=${3:-1}
```

Comment:
How about 4 lines in a row for $1 through $4, then the echo "Taking bias ...

And a comment for those unfamiliar with bash details that the default value if the 3rd param isn't supplied is 1, not -1 which one might suspect.